statsmodels/genmod/families/family.py

```diff
@@ -1451,10 +1455,28 @@ def loglike_obs(self, endog, mu, var_weights=1., scale=1.):
 
         Notes
         -----
-        This is not implemented because of the complexity of calculating an
-        infinite series of sums.
+        If eql is True, the Extended Quasi-Likelihood is used.  At present,
+        this method returns NaN if eql is False.  When the actual likelihood
+        is implemented, it will be accessible by setting eql to False.
+
+        References
+        ----------
+        http://support.sas.com/documentation/cdl/en/stathpug/67524/HTML/default/viewer.htm#stathpug_hpgenselect_details16.htm
         """
-        return np.nan
+        if self.eql == False:
```

Comment:
Also has this been PEP8'ed? I recall that its preferred just to do a `if not self.eql:` but I have no clue what the rule is... But I feel like I've seen it somewhere...